### PR TITLE
enh: UI/UX improvements

### PR DESCRIPTION
**Pull request by cdtinney**
This PR makes a bunch of small changes, e.g.
* Wrap preset breakdown columns so that they aren't hidden when 3 columns exist in a smaller window.
* Fix tabindex for breakdown list so you can tab between the inputs.
* Add Copy Image to Clipboard buttons for both preset and breakdown, so you can paste directly into Imgur.
* Prevent unnecessary scrollbar in the item dialog when the dropdown is opened.

### CODE REVIEW

File: src/utility/export-to-png.ts [hunk 1]
```diff
@@ -7,6 +7,31 @@ export const exportAsImage = async (element: HTMLElement | null, preface: string
...
+
+export const copyImageToClipboard = async (element: HTMLElement | null): Promise<void> => {
+  if (!element) {
+    return;
+  }
+
+  await createCanvas(element, (_imageBlobURL: string, canvas: HTMLCanvasElement) => {
+      canvas.toBlob((blob) => {
+        if (!blob) {
+          // TODO: Failure snackbar?
```
Comment (nullopt) [inline src/utility/export-to-png.ts:23]: What's this?

File: src/utility/export-to-png.ts [hunk 2]
```diff
@@ -7,6 +7,31 @@ export const exportAsImage = async (element: HTMLElement | null, preface: string
...
+          return;
+        }
+        
+        console.error(blob);
+        const item = new ClipboardItem({ 'image/png': blob });
```
Comment (nullopt) [inline src/utility/export-to-png.ts:28]: `ClipboardItem` API is not supported by Firefox

File: src/components/PresetEditor/PresetEditor.tsx [hunk 3]
```diff
@@ -74,6 +74,10 @@ export const PresetEditor = () => {
     await exportAsImage(exportRef.current, `PRESET_${name.replaceAll(" ", "_")}`);
   }, [name]);
 
+  const onCopyToClipboard = useCallback(async () => {
+    await copyImageToClipboard(exportRef.current);
+  }, [name]);
```
Comment (nullopt) [inline src/components/PresetEditor/PresetEditor.tsx:79]: I don't think the `name` needs to be in the dependency list

**Review by nullopt — CHANGES_REQUESTED** [review]

File: src/utility/export-to-png.ts (hunk 1 above)
Comment (cdtinney) [reply src/utility/export-to-png.ts:23]: I was going to add a failure toast (snackbar?), but can just log it or ignore it?

Comment (nullopt) [reply src/utility/export-to-png.ts:23]: Yeah, I think it's worth adding a snackbar if it errors, as everything else has one

**Review by nullopt — CHANGES_REQUESTED** [review]

**Pull request merged**